### PR TITLE
Add test week instruction

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

**Comment by adamruzicka** [comment]
Shouldn't this be opened against theforeman/tool_belt?

File: procedures/foreman/release.md.erb [hunk 1]
```diff
@@ -78,6 +78,9 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 <% end %>- [ ] Announce the release on [Discourse](https://community.theforeman.org/c/release-announcements/8)
 - [ ] Update the topic in #theforeman channel on Freenode
 - [ ] Share the release announcement on twitter
+<% if is_rc %>
+- [ ] Instruct lzap to initiate Test Week on Discourse (only applies to RC1)
```
Comment (ekohl) [inline procedures/foreman/release.md.erb:82]: https://github.com/theforeman/tool_belt/blob/4f126b787113c000c9d7b0cee186533e837baa7b/procedures/foreman/release.md.erb#L112 has a piece of code which checks for RC1 in code. I'd prefer that over a comment.

**Comment by lzap** [comment]
> Shouldn't this be opened against theforeman/tool_belt?

Da fak. I am trying Github included editor feature.

**Comment by lzap** [comment]
Okay, also I have a nice URL: https://github.com/theforeman/tool_belt/pull/404

**Pull request closed without merging**